data_expansion2/localized/de_patch/text/conversations/px2_05_wildernesses/px2_05_si_frontal_assault.stringtable

```diff
@@ -322,7 +322,7 @@ Die überraschende Geschwindigkeit hilft euch. Ihr rennt mit dem Rammbock auf da
     </Entry>
     <Entry>
       <ID>68</ID>
-      <DefaultText>Du hast einen Atemzug Zeit, um zu entscheiden, welchem deiner Gefährten du zur Seite springst&#160;…</DefaultText>
```

Comment:
Also in Bayern hab ich das noch nie gehört. *lach* Aber wie gesagt, ich kann damit lebenn.